src/Contract/Rector/RectorInterface.php

```diff
@@ -21,7 +20,16 @@ public function getNodeTypes(): array;
 
     /**
      * Process Node of matched type
-     * @return Node|Node[]|null|NodeTraverser::*
+     * @return Node|Node[]|null|int
+     *
+     * For int return, choose:
+     *
+     *   ✔️ To decorate current node and its children to not be traversed on current rule, return one of:
+     *          - NodeVisitor::DONT_TRAVERSE_CHILDREN
+     *          - NodeVisitor::DONT_TRAVERSE_CURRENT_AND_CHILDREN
+     *
+     *   ✔️ To remove node of Node\Stmt or Node\Param, return:
+     *          - NodeVisitor::REMOVE_NODE
      */
```

Comment:
I prefer to keep original list, as now any `int` value can be returned by PHPStan, which is not true.
Could you revert this one?

Reply:
Ok, the rule classes itself then need update to add `@return` for specific constant then, eg:

```diff
    /**
     * @param StmtsAwareInterface $node
+    * @return null|Node|NodeVisitor::DONT_TRAVERSE_CHILDREN
     */
    public function refactor(Node $node): null|Node|int
```

I will revert this and update rule classes

Reply:
I created PR to use `NodeVisitor::*` on `RectorInterface::refactor()`, then update its rules `@return` to use its constant at PR:

- https://github.com/rectorphp/rector-src/pull/7409